apps/web/app/globals.css

```
@@ -117,7 +116,18 @@ h4 {
 }
 
 .fade-in-up {
-  @apply animate-fadeInUp;
+  animation: fadeInUp 1s ease forwards;
+}
+
+@keyframes fadeInUp {
```

Stylelint (17.13.0)

[error] Expected keyframe name "fadeInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)